src/main.py

```diff
@@ -83,6 +85,17 @@ async def post_message(chat_id: str, message: Chat):
     return {"message": ai_message["text"]}
 
 
+@app.post("/chat")
+async def post_message(message: Chat):
```

Comment:
We already have a route with the same name

Reply:
`/ask` sounds better, right?